src/main/java/com/synopsys/integration/alert/workflow/scheduled/PhoneHomeTask.java

```diff
@@ -128,6 +131,20 @@ public String scheduleCronExpression() {
         return CRON_EXPRESSION;
     }
 
+    public Map<String, String> getPhoneHomeVariables() {
+        Map<String, String> environment = System.getenv();
+        String skipPhoneHome = environment.get(PhoneHomeClient.SKIP_PHONE_HOME_VARIABLE);
+        String phoneHomeURLOverride = environment.get(PhoneHomeClient.PHONE_HOME_URL_OVERRIDE_VARIABLE);
+        Map<String, String> phoneHomeVariables = new HashMap<>();
+        if (StringUtils.isNotBlank(skipPhoneHome)) {
+            phoneHomeVariables.put(PhoneHomeClient.SKIP_PHONE_HOME_VARIABLE, skipPhoneHome);
+        }
+        if (StringUtils.isNotBlank(phoneHomeURLOverride)) {
+            phoneHomeVariables.put(PhoneHomeClient.PHONE_HOME_URL_OVERRIDE_VARIABLE, phoneHomeURLOverride);
+        }
+        return phoneHomeVariables;
+    }
```

Comment:
If we're passing a map regardless, we might as well just pass the entire System.getenv() map and let Phonehome deal with it. This seems like unnecessary work.
If your goal is to allow this method for testing purposes, then just return System.getenv() here.

Reply:
I think we want to use BlackDuckPhoneHomeHelper here.  We also get the black duck data for phone home so that might be what we want to use here.